server/middleware/validation.js

```diff
@@ -91,7 +91,14 @@ const validate = {
       .isEmpty({ ignore_whitespace: true })
       .withMessage('AuthorID can not be left empty: Please input amount')
       .isInt()
-      .withMessage('AuthorID is not valid integer: Please input a valid authorID'),
+      .withMessage('AuthorID is not valid integer: Please input a valid authorID')
+      .custom( async (id) => {
+        const isExist = await util.checkAuthorID(id);
+        if (!isExist) {
+          throw new Error('No author with the specified ID was found')
```

Comment:
Why are you throwing and is this error caught somewhere?

Reply:
https://express-validator.github.io/docs/custom-validators-sanitizers.html

Reply:
That is how to write express validator custom function. it would catch the error its self.